### PR TITLE
rewrite the architecture decision for exposing bouncer on port 80

### PR DESCRIPTION
**Pull request by fredericfran-gds**

### CODE REVIEW

File: doc/architecture/decisions/0035-bouncer-load-balancer-on-port-80-and-443.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Consequences
+
+In order to implement the decision above, the following code changes were done:
+1. the terraform [plan](terraform/projects/infra-public-services/main.tf)
+was changed so that the input to the `aws/lb` terraform module specifies a second
+listener `"HTTP:80"   = "HTTP:80"` in the listener map for the Bouncer public
+load balancer.
+
+2. the `aws/lb` terraform module [here](terraform/modules/aws/lb/main.tf) had
```
Comment (deanwilson) [inline doc/architecture/decisions/0035-bouncer-load-balancer-on-port-80-and-443.md]: The link doesn't work.

File: doc/architecture/decisions/0035-bouncer-load-balancer-on-port-80-and-443.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Date: 2018-10-22
+
+## Status
+
+Pending
+
+## Context
+
+This architecture decision record is part of the GOV.UK migration.
```
Comment (deanwilson) [inline doc/architecture/decisions/0035-bouncer-load-balancer-on-port-80-and-443.md]: This isn't  required, it's in the repo so it's concerning the repo.

File: doc/architecture/decisions/0035-bouncer-load-balancer-on-port-80-and-443.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+Currently, Bouncer public load balancer accepts HTTPS request on port 443 and
+redirect the traffic to port 80 on the Bouncer servers.
```
Comment (deanwilson) [inline doc/architecture/decisions/0035-bouncer-load-balancer-on-port-80-and-443.md]: `redirects`

File: doc/architecture/decisions/0035-bouncer-load-balancer-on-port-80-and-443.md (hunk 1 above)
Comment (fredericfran-gds) [reply doc/architecture/decisions/0035-bouncer-load-balancer-on-port-80-and-443.md]: will fix this

File: doc/architecture/decisions/0035-bouncer-load-balancer-on-port-80-and-443.md (hunk 2 above)
Comment (fredericfran-gds) [reply doc/architecture/decisions/0035-bouncer-load-balancer-on-port-80-and-443.md]: will fix

File: doc/architecture/decisions/0035-bouncer-load-balancer-on-port-80-and-443.md (hunk 3 above)
Comment (fredericfran-gds) [reply doc/architecture/decisions/0035-bouncer-load-balancer-on-port-80-and-443.md]: will fix

**Comment by fredericfran-gds** [comment]
Great!

**Pull request merged**